web/src/components/atomic/DocsLink.vue

```diff
@@ -1,32 +1,22 @@
 <template>
-  <a :href="`${docsBaseUrl}${url}`" target="_blank" class="text-blue-500 hover:text-blue-600 cursor-pointer mt-1"
+  <a :href="`${docsUrl}`" target="_blank" class="text-blue-500 hover:text-blue-600 cursor-pointer mt-1"
     ><Icon name="question" class="!w-4 !h-4"
   /></a>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
+<script lang="ts" setup>
```

Comment:
unrelated refactor ?!?

Reply:
@6543 Its a "newer" way of writing components in vue 3 which has much nice IDE support IMO. I would just update it each time I touch a component.